scripts/leetcode.js

```diff
@@ -81,46 +81,44 @@ const upload = (token, hook, code, problem, filename, sha, commitMsg, cb = undef
     });
 };
 
-const getAndInitializeStats = (problem) => {
-  return chrome.storage.local.get('stats')
-    .then(({ stats }) => {
-      if (stats == null || stats === {}) {
-        // create stats object
-        stats = {};
-        stats.solved = 0;
-        stats.easy = 0;
-        stats.medium = 0;
-        stats.hard = 0;
-        stats.shas = {};
-      }
+const getAndInitializeStats = problem => {
+  return chrome.storage.local.get('stats').then(({ stats }) => {
+    if (stats == null || stats === {}) {
+      // create stats object
+      stats = {};
+      stats.solved = 0;
+      stats.easy = 0;
+      stats.medium = 0;
+      stats.hard = 0;
+      stats.shas = {};
+    }
 
-      if (stats.shas[problem] == null) {
```

Comment:
Space are considered as a code change and its difficult to do Code review. Could you please resubmit without these changes

Reply:
Yes, perhaps try only making the change related to the issue #12 fix, but don't run npm prettier on the code

Reply:
@arunachalamev @bjpfox Reverted the prettier changes, please have a look.

Reply:
Looks good to me. I have also tested locally and can confirm everything works fine now. Great job 👍 

@arunachalamev I think this is good to merge, if you're happy to do so?

Reply:
It is working for a new repository but not working for the repo which was used previously, so maybe some issues in linking

Reply:
@Prakhart-debug These changes doesn’t relate anything towards repository handling.

Can you share any console logs while commiting to the old repo?

Reply:
@Prakhart-debug I tested it using my previously used repo, and it worked perfectly.

After you installed / reloaded the chrome extension did you specify the same repo as previously used? for me, after I did that, everything worked.